packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -87,6 +87,24 @@
   padding-inline-start: var(--PaddingInline);
 }
 
+.clear {
+  display: none;
+  padding-inline-start: var(--PaddingInline);
+}
+
+// stylelint-disable-next-line selector-max-type
```

Comment:
What's better? Keeping the styles in one place and ignoring the rule. Or creating a new variant of `Button` with the correct css?

Reply:
If the Password and Copy buttons are going to match Clear, I think it'd be worth creating a new `Button` variant. Would that remove the need for `.clear`?

Reply:
What if we added a second class to `.prefix/.suffix` when we want this behavior? Something like:

```html
<div class="root">
  <span class="prefix" aria-hidden="true">
    <svg>...</svg>
  </span>
  <input class="input" type="search" value="Search Planets">
  <span class="suffix affix-hidden">
    <button variant="affix">x</button>
  </span>
</div>
```

This way, `Input` controls the behavior and styles specific to itself, and Button only brings what it needs to be a Button.

Reply:
I think that would work. Lemme give it a try

Reply:
@edburyenegren-okta I've updated the PR with your suggestions. I think it looks better now. 
So I'll just add an `affix` variant in `Button` for the button styles?

Reply:
Sounds good! We can re-use that Button for Password and Click-to-Copy as well.